.github/workflows/codeql.yml

```
@@ -10,8 +10,16 @@ on:
   pull_request:
     branches: [ "master", "main" ]
     paths:
-      - '**.py'
+      - 'app.py'
+      - 'csr.py' 
+      - 'start_server.py'
+      - 'session_crypto.py'
+      - '_version.py'
       - '.github/workflows/codeql.yml'
+    paths-ignore:
```

actionlint (1.7.7)

both "paths" and "paths-ignore" filters cannot be used for the same event "pull_request". note: use '!' to negate patterns

(events)